packages/tocco-ui/src/EditableValue/typeEditors/DateAbstract.js

```diff
@@ -72,24 +71,18 @@ class DateAbstract extends React.Component {
     return null
   }
 
-  componentWillReceiveProps(props) {
-    this.handleButtonVisibility()
-    const locale = this.getLocale(props.intl.locale)
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    const locale = this.getLocale(this.props.intl.locale)
 
     if (this.Flatpickr && this.flatpickr) {
       this.Flatpickr.localize(locale)
       this.flatpickr.set('locale', locale)
 
-      this.flatpickr.set('altFormat', props.options.flatpickrOptions.altFormat)
-      this.flatpickr.setDate(props.value, false)
+      this.flatpickr.set('altFormat', this.props.options.flatpickrOptions.altFormat)
```

Comment:
you removed functionality. Open the story and try editing the Value via knobs. The value will not change...